tasks/apt_install.yml

```diff
@@ -0,0 +1,71 @@
+- name: Add ruby repo
+  apt_repository:
+    repo: 'ppa:brightbox/ruby-ng'
+
+- name: Add yarn gpg repo key
+  apt_key:
+    url: "https://dl.yarnpkg.com/debian/pubkey.gpg"
+
+- name: Add yarn repo
+  apt_repository:
+    repo: "deb https://dl.yarnpkg.com/debian/ stable main"
+
+- name: Add Go repo
+  apt_repository:
+    repo: 'ppa:gophers/archive'
+
+- name: Remove ruby1.8
+  apt:
+    name: ruby1.8
+    state: absent
+
+- name: Add nodejs gpg key
+  apt_key:
+    url: https://deb.nodesource.com/gpgkey/nodesource.gpg.key
+
+- name: Add nodejs repositoriy
+  apt_repository:
+    repo: "deb https://deb.nodesource.com/node_8.x artful main"
```

Comment:
With all those repos to add, why not use one task and `with_items`?

Reply:
fixed